apps/console/package.json

```
@@ -34,6 +34,7 @@
         "start:verbose": "npm run start -- --env LOG_LEVEL=verbose"
     },
     "dependencies": {
+        "@wso2is/access-control": "^1.0.0",
```

This has to be same as version in `lerna.json`. FYI

```suggestion
        "@wso2is/access-control": "^1.2.227",
```